README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Little bit about me.
+
+Hello my name is Arturo Juarez. I am studying **International Relations** under the Committee of International Relations at the *University of Chicago*. I was born in California but I grew up in both Illinios and and Southern California. As such I consider myself both a Midwesterner and a Californian. My interests incluce reading, outdoor running, and traveling to new destinations both domestically and internationally. 
+
+### More about me
+
+My *hobbies* include playing sports such as basketball, baseball, and volleyball. I enjoy the **outdoors** as much as I enjoy spending a lazy day **indoors**. I care deeply about the social, economic, and civil challenges facing our democracy today and I actively donate my time, energy, and resources to many causes I find of pressing importance. I *source* my news at times from [link](https://www.npr.org/) [link](https://www.washingtonpost.com/) or [link](https://thehill.com/) 
+
+#### Lists
+
+* Unordered List
+* Red Fruit
+  + strawberries 
+  + apples 
+  + raspberries 
+  
+  1. Ordered List
```

Comment:
This isn't a huge deal, but I don't think you have to say "ordered list" and "unordered list" since which kind it is comes through in the formatting.

Reply:
I think the insertion of the ordered list here confuses me a bit. What is its purpose?

Reply:
Notice that the the beginning of the ordered list is indented so that it is in line with the previous bullet point rather than the left margin. In order to start a new list or paragraph, make sure to end the previous line with **two spaces**